### ISSUE TITLE
Intégrer les stats dans les pages jeux de données

### ISSUE THREAD

**Issue by streino** [opendatateam/udata-front-kit#377]
Météo-France a l'air intéressé également

![image](https://github.com/opendatateam/udata-front-kit/assets/30210525/92682670-6e94-45a0-a3da-10e0a2c40df7)

**Comment by streino** [comment]
@nicolaskempf57 c'est déjà dispo dans data.gouv.fr-components ?

**Comment by nicolaskempf57** [comment]
> @nicolaskempf57 c'est déjà dispo dans data.gouv.fr-components ?

Non, le composant est toujours dans [udata-front](https://github.com/etalab/udata-front/blob/master/udata_front/theme/gouvfr/assets/js/components/charts/chart.vue).

On commence seulement les premières requêtes de metrics sur le front avec la nouvelle admin, sans le composant. 

Quand on aura plus de recul sur les requêtes pour proposer un fonctionnement stable, on pourra tout mettre dans `data.gouv.fr-components`.

**Comment by johanricher** [comment]
C'est un besoin également partagé pour le portail culture (intégration de stats / métriques sur la page d'accueil, à l'instar de celle de data.gouv.fr).

**Comment by johanricher** [comment]
Les dataviz metrics sont désormais componentisées et peuvent être utilisées par udata-front-kit : https://github.com/datagouv/cdata/blob/main/pages/organizations/%5Boid%5D/information.vue#L25

**Comment by nicolaskempf57** [comment]
> Nouvelle version sur datagouv en encart:

Le composant permet bien de faire les 2, la distinction se fait sur la propriété `size`
